src/overtone/inst/sampler.clj

```diff
@@ -28,14 +28,16 @@
 ; s.sendMsg("/b_close", 2); // close the file.
 ; s.sendMsg("/b_free", 0); // frees the buffer
 
-(defsynth granular [out 0, buf 0, pan 0, start 0, amp 0.8, dur 0.25]
+;; TODO: Invalid ugen rate
+#_(defsynth granular [out-bus 0, buf 0, pan 0, start 0, amp 0.8, dur 0.25]
   (let [grain (play-buf 1 buf (buf-rate-scale:kr buf) 1
                         (* start (buf-frames:ir buf)) 0)
         env (- (env-gen:ar (perc 0.01 dur) 1 1 0 1 FREE) 0.001)]
-    (out out (* (pan2 (* grain env) pan) amp))))
+    (out out-bus (* (pan2 (* grain env) pan) amp))))
 
-(defn test-flute []
-  (defonce flute (load-sample (load-sample "/home/rosejn/projects/overtone/instruments/samples/flutes/flutter-flute-1.wav")))
-  (hit flute :dur 2.0))
+;; TODO: hit is not defined
+#_(defn test-flute []
+    (defonce flute (load-sample (load-sample "/home/rosejn/projects/overtone/instruments/samples/flutes/flutter-flute-1.wav")))
```

Comment:
we could replace this with freesound url eventually

Reply:
Yes o/